.github/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+# Pull Request
+
+## Overview
+
+- **Title**: [Descriptive title of the changes]

```

Comment:
add description, after title

Reply:
Updated. Please verify this is what you described.